rero/rero-ils-ui
# persons: Implements admin detail view

## Why are you opening this PR?

- Task 1139: Implement detail view

## How to test?
- Use temporary patch: `https://github.com/Garfield-fr/rero-ils/tree/zub-api-person-record`
- Show person detail view
- Check all actions are disabled

## Code review check list


Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
